src/raiden_libs/transport/transport.py

```diff
@@ -19,7 +19,7 @@ class Transport(gevent.Greenlet):
     """
     def __init__(self):
         super().__init__()
-        self.message_callbacks = list()
+        self.message_callbacks: list = list()
```

Comment:
While `list` works, we always try to use classes from the `typing` module as annotations. So `typing.List` here.

Reply:
` self.message_callbacks: typing.List = list()` ?

Reply:
yes, or import list from typing first and then just use it there